Two logo functionality

aCyborg opened the pull request:
This adds the two logo option into the reveal base template and updates the readme with instructions. This also general updates some of the code blocks in the readme

aCyborg committed:
add two logo functionality

aCyborg committed:
Update readme to be styled properly.

Crell commented on README.md:
```diff
@@ -14,7 +14,7 @@ Use this repository when creating a slide deck to present at conferences if you
 3. If you plan to write or modify the sass, install breakpoint: http://breakpoint-sass.com/
 4. Update the Title, Author, and Description metadata in the head of the index.html file.
-4. Edit the html within the index file to contain the content for your presentation. Place your slide content within "&#060;div class="slides"&gt; Your Slides Here! &#060;/div&gt;".
+4. Edit the html within the index file to contain the content for your presentation. Place your slide content within "<div class="slides"> Your Slides Here! </div>".
```
This change seems unrelated. Does it break Markdown rendering?  Editor goof?

Crell commented on README.md:
```diff
@@ -88,29 +86,45 @@ This provides the option of having 2 column, 3 column, or 4 column layouts on yo
+  1. Open your index.html file in the root.
+  2. On line 404 there is example markup, commented out for this additional logo, or you can use this example.
+
+            <a class="logo-wrapper" href="url-to-partner.com" title="Link to Palantir.net's website">
```
The example title should probably be a link to the partner's website.

aCyborg committed:
update readme styles

aCyborg committed:
update readme

example copy for the secondary/partner logo has been updated to show "Partner's" website rather than "palantir's"

Crell commented:
Thanks, @aCyborg!
